packages/components/root/src/root.css

```diff
@@ -110,7 +111,7 @@
 	grid-column: 1 / 1;
 }
 :host ::slotted(.sidebar-right),
-:host > .content ::slotted(.sidebar-inner-right) {
+.content ::slotted(.sidebar-inner-right) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the unknown "slotted" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aunknown-pseudo)